frontend/src/match.js

```diff
@@ -0,0 +1,29 @@
+// @Front-end: Call startMatch() on form submit
+/** Add participant, find match if possible, keep searching for match every 5 seconds */
+function startMatch() {
+  addParticipant();
+  searchMatch();
+}
+
+/** Add participant, find match with existing participants if possible */
+function addParticipant() {
+  const params = new URLSearchParams();
+  fetch('/add-participant', {method: 'POST', body: params});
+}
+
+/** Search for match in list of all matches */
+function searchMatch() {
+  fetch('/search-match').then(response => response.json()).then((match) => {
+    // No match yet
+    if (match.id == 0) {
```

Comment:
What do you think about sending back a `matchStatus` which is true or false based on whether a match was found or not?